R/package.R

```diff
@@ -30,6 +30,17 @@
 NULL
 
 .onLoad <- function(...) {
+  # Points to specific Python installation if using
+  # RStudio inside the Databricks cluster environment
+  db_path <- "/databricks/python/bin/python"
+  if(file_exists(db_path)) {
+    options(
+      reticulate.virtualenv.starter = c(
```

Comment:
I don't think `/databricks/python/bin/python` can be a virtualenv starter - it itself is already a virtualenv derived from `/usr/bin/python3`. It should be a non-issue however because reticulate will call `normalizePath()` which will resolve the symlink `databricks/python/bin/python -> /usr/bin/python`, and cause reticulate to use the system python, which it would have done already in this situation.